benchmarks/inference-server/text-generation-inference/main.tf

```diff
@@ -84,7 +84,6 @@ resource "kubernetes_manifest" "hpa-cpu" {
 }
 
 resource "kubernetes_manifest" "tgi-pod-monitoring" {
-  count = local.custom_metrics_enabled ? 1 : 0
```

Comment:
Should we decouple the yaml files from the same directory as well? I guess I'm thinking maybe it makes sense to move the podmonitoring yaml to either a separate monitoring folder or the existing tgi manifests folder to indicate its always deployed.

Reply:
Yes, that's a good idea. 

I kept the DCGM PodMonitoring in hpa-templates because that one is specific for HPA (due to the metric renaming hack)